main.tf

```diff
@@ -12,32 +12,46 @@ resource "helm_release" "gitlab_runner" {
   namespace        = var.namespace
   version          = var.chart_version
   create_namespace = var.create_namespace
+  atomic           = true
 
 
   values = [
-    yamlencode(merge({
+    yamlencode({
+
       image                   = var.runner_image
       gitlabUrl               = var.gitlab_url
       concurrent              = var.concurrent
       runnerRegistrationToken = var.runner_registration_token
+      replicas                = var.replicas
```

Comment:
https://gitlab.com/gitlab-org/charts/gitlab-runner/-/blob/main/values.yaml#L36

We might to do some variable validation to ensure replicas are not used with `runnerToken`

Reply:
runnerToken is not defined in this project. i can include it and use a derivative value for replicas
`locals {
      replicas = var.runner_token != null ? 1 : var.replicas
}`